packages/cli/tests/create-tutorial.test.ts

```diff
@@ -216,20 +193,39 @@ test('cannot eject on an astro project that is not using TutorialKit 2', async (
   `,
   );
 
-  await execa('npm', ['install'], {
+  await execa('pnpm', ['install'], {
     cwd: dest,
   });
 
   await expect(
     execa('node', [cli, 'eject', name, '--force', '--defaults'], {
       cwd: tmpDir,
-      env: {
-        TK_DIRECTORY: baseDir,
-      },
     }),
   ).rejects.toThrow(`Could not find import to '@tutorialkit/astro'`);
 });
 
 function normaliseSlash(filePath: string) {
   return filePath.replace(/\\/g, '/');
 }
+
+async function runPnpmInstall(dest: string, baseDir: string) {
+  const packageJsonPath = path.join(dest, 'package.json');
+  const packageJson = JSON.parse(await fs.readFile(packageJsonPath, 'utf-8'));
+
+  packageJson.pnpm = {
+    overrides: {
+      '@astrojs/language-server': '2.11.1',
```

Comment:
This is the version that we were using before that didn't exhibit that issue. I still think that doing the `file:..` thing below is wrong. I'm wondering if moving to use `.tar.gz` files would solve the problem because then their `node_modules` would be entirely managed by the package manager.